src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingGuards.java

```diff
@@ -200,6 +219,14 @@ public void onUpgradeComplete(final int newLevel)
     }
 
     /**
+     * Sets the player to follow.
+     *
+     * @param player the player to follow.
+     */
+    public void setPlayerToFollow(final EntityPlayer player)
+    {
+        this.followPlayer = player;
+    }    /**
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)